wiki/en/Tips-Tricks-More.md

```diff
@@ -29,7 +33,7 @@ You can also use [VoiceMeeter](https://www.vb-audio.com/Voicemeeter/banana.htm)
 
 ## Recording Jamulus on Windows with Reaper
 
-Jamulus user [Rob Durkin](https://sourceforge.net/u/bentwrench/profile/) has written a [guide to recording the output of Jamulus](https://docs.google.com/document/d/1tENfNKTWHasuTg33OdLLEo4-OOhWJolOo42ffSARxhY/edit) (Google Doc) using the ReaRoute add-on for [Reaper](https://www.reaper.fm/). 
+Jamulus user [Rob Durkin](https://sourceforge.net/u/bentwrench/profile/) has written a [guide to recording the output of Jamulus](https://docs.google.com/document/d/1tENfNKTWHasuTg33OdLLEo4-OOhWJolOo42ffSARxhY/edit) (Google Doc) using the ReaRoute add-on for [Reaper](https://www.reaper.fm/).
 
 ## Remote management of recordings
 
```

Comment:
This document wants a clear split between client and server.  Almost all of it is client.

Reply:
Issue https://github.com/jamulussoftware/jamuluswebsite/issues/730 raised.